test/notifications_test.js

```diff
@@ -81,29 +86,47 @@ describe('notifications', function() {
 
   serverManager.inject();
 
-  beforeEach(function*() {
-    mq.connect(mqSocket);
-    yield chakram.post(subscriptionsUrl, subscription);
-  });
+  describe('no subscription registered', function() {
+    beforeEach(function() {
+      // Start up database connection. It will be closed once the DB is deleted
+      require('../dao/database').init(serverManager.profilePath);
```

Comment:
Mmm I wonder if we shouldn't close the DB explicitely in afterEach; what do you think ?